cli/internal/api/api.go

```diff
@@ -74,8 +76,8 @@ func (c *Client) GetLogs(ctx context.Context, workflowName string) (responses.Ge
 	return output, nil
 }
 
-// StreamLogs streams the logs of a workflow.
-func (c *Client) StreamLogs(ctx context.Context, w io.Writer, workflowName string) error {
+// StreamLogs streams the logs of a workflow starting after loggedBytes.
+func (c *Client) StreamLogs(ctx context.Context, w io.Writer, workflowName string, loggedBytes int64) error {
```

Comment:
shouldn't we use unsigned int? or this method should validate (>= 0)

Reply:
also maybe 'skipFirstBytes' or similar name would better express what this parameter is for?